src/main/java/message_view/application_business_rule/ViewMessageOutputboundary.java

```
@@ -0,0 +1,7 @@
+package message_view.application_business_rule;
+
+public interface ViewMessageOutputboundary {
+    ViewMessageResponseModel prepareSuccessView(ViewMessageResponseModel response);
+
+    ViewMessageResponseModel prepareFailView(String error);
```

You need to throw an error with the fail view